### PR TITLE
Update for obsoletions

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to dotnet/docs#66292

### CODE REVIEW

File: xml/System.Net.Security/EncryptionPolicy.xml [hunk 1]
```diff
@@ -119,7 +118,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>Allow no encryption and request that a NULL cipher be used if the other endpoint can handle a NULL cipher.</summary>
+        <summary>Allow no encryption and request that a NULL cipher be used if the other endpoint can handle a NULL cipher. Starting in .NET 7, this field is obsolete.</summary>
```
Comment (rzikm) [inline xml/System.Net.Security/EncryptionPolicy.xml:121]: is "field" the right word here? I would use "value" personally, but if we call it field consistently in other docs, feel free to ignore this comment.

Comment (gewarren) [reply xml/System.Net.Security/EncryptionPolicy.xml:121]: Good question...it sounded a little weird to me too. What about using "member" instead? I'm not sure about "value" - is that commonly used when talking about enum members?

One reason to use "field" is that the heading in docs is "Fields", for example: https://docs.microsoft.com/en-us/dotnet/api/security.sslprotocolside?view=xamarin-ios-sdk-12#fields.

Comment (wfurt) [reply xml/System.Net.Security/EncryptionPolicy.xml:121]: It does seems like all the enum description is under "fields" header
https://docs.microsoft.com/en-us/dotnet/api/system.net.security.encryptionpolicy?view=net-6.0

General documentation seems to use terms "members" when talking about the set and "value" when talking about specific one
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum

Comment (gewarren) [reply xml/System.Net.Security/EncryptionPolicy.xml:121]: The C# spec says "Each enum member has an associated constant value." So I'll go with "member" in this case. Might be worth modifying the name of the heading from "Fields" to "Members" too cc @mimisasouvanh.

File: xml/System.Net.Security/EncryptionPolicy.xml [hunk 2]
```diff
@@ -83,7 +82,7 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>Prefer that full encryption be used, but allow a NULL cipher (no encryption) if the server agrees.</summary>
+        <summary>Prefer that full encryption be used, but allow a NULL cipher (no encryption) if the server agrees. Starting in .NET 7, this field is obsolete.</summary>
```
Comment (gewarren) [inline xml/System.Net.Security/EncryptionPolicy.xml:85]:
```suggestion
        <summary>Prefer that full encryption be used, but allow a NULL cipher (no encryption) if the server agrees. Starting in .NET 7, this member is obsolete.</summary>
```

File: xml/System.Net.Security/EncryptionPolicy.xml (hunk 1 above)
Comment (gewarren) [inline xml/System.Net.Security/EncryptionPolicy.xml:121]:
```suggestion
        <summary>Allow no encryption and request that a NULL cipher be used if the other endpoint can handle a NULL cipher. Starting in .NET 7, this member is obsolete.</summary>
```

**Review by rzikm — APPROVED** [review]
LGTM

**Pull request merged**